doc-files/release-notes-24.md

```diff
@@ -0,0 +1,223 @@
+# Release Notes for JavaFX 24
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+These release notes cover the standalone JavaFX 24 release. JavaFX 24 requires JDK 22 or later. JDK 24 is recommended.
+
+## Important Changes
+
+### JavaFX 24 Requires JDK 22 or Later
+
+JavaFX 24 is compiled with `--release 22` and thus requires JDK 22 or later in order to run. If you attempt to run with an older JDK, the Java launcher will exit with an error message indicating that the `javafx.base` module cannot be read.
+
+See [JDK-8340003](https://bugs.openjdk.org/browse/JDK-8340003) for more information.
+
+### JavaFX Applications Must Use `--enable-native-access`
+
+Running a JavaFX application on JDK 24 will produce a warning from each of the three JavaFX modules that rely on native access, due to the changes specified in [JEP 472](https://openjdk.org/jeps/472). Each warning will include the following message:
+
+```
+WARNING: Restricted methods will be blocked in a future release unless native access is enabled
+```
+
+In order to suppress the warning now, and to be able to run your application at all in a subsequent version of the JDK, you need to explicitly enable native access for all modules that need it. This is done by passing `--enable-native-access=<list-of-modules>` to `java` on the command line, listing the modules that you grant native access. This list of modules includes `javafx.graphics` and, optionally, `javafx.media` and `javafx.web`, if your application uses those modules.
+
+For example:
+
+```
```

Comment:
minor: you could use 'console' language identifier for syntax highlighting of command lines, though it does not do much for the code blocks in this file

preview the result here:
https://github.com/andy-goryachev-oracle/Test/blob/main/doc/Tests/Test.md

Reply:
Interesting idea, thanks. I think I'll leave this as is, since, as you point out, it doesn't really do much in this particular case.